ipyvolume/embed.py

```diff
@@ -140,63 +57,118 @@ def save_js(dirname, makedirs=True):
     src = os.path.join(dir_name_src, "index.js")
     shutil.copy(src, dst)
```

Comment:
cc @jasongrout
If we want this to go upstream, we need to do this differently. Scan the while state for all module names, and versions, and fetch them from a CDN, and run require.config() so it knows where to fetch them. Then there is the issue with multiple versions of the same package.. but maybe that's not sth we want to support initially.